operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -212,12 +220,17 @@ func NewDatacenterCondition(conditionType DatacenterConditionType, status corev1
 	}
 }
 
+// Note: The below property 'SuperUserUpserted' needs to be refactored 
+// at some point to treat superuser as one word instead of two. Also, 
+// this is not just for the one superuser any more, but all users
+// defined on the CassandraDatacenter resource.
```

Comment:
I didn't want this comment dumped in the CRD, so I put it up here instead.

Reply:
I think you can reset the status bits for the user creation. Make whatever is ideal for you, and label the existing stuff deprecated.